src/components/pages/CompanyPage/dummy/chart.json

```diff
@@ -0,0 +1 @@
date":"2017-12-19","color":"#ebedf0","total":0},{"date":"2017-12-20","color":"#ebedf0","total":0},{"date":"2017-12-21","color":"#ebedf0","total":0},{"date":"2017-12-22","color":"#ebedf0","total":0},{"date":"2017-12-23","color":"#ebedf0","total":0}]},{"days":[{"date":"2017-12-24","color":"#ebedf0","total":0},{"date":"2017-12-25","color":"#ebedf0","total":0},{"date":"2017-12-26","color":"#ebedf0","total":0},{"date":"2017-12-27","color":"#ebedf0","total":0},{"date":"2017-12-28","color":"#ebedf0","total":0},{"date":"2017-12-29","color":"#ebedf0","total":0},{"date":"2017-12-30","color":"#ebedf0","total":0}]},{"days":[{"date":"2017-12-31","color":"#ebedf0","total":0}]}]}},{"year":2018,"totalIssueContributions":0,"totalCommitContributions":0,"totalRepositoryContributions":0,"totalPullRequestContributions":0,"totalPullRequestReviewContributions":0,"totalRepositoriesWithContributedIssues":0,"totalRepositoriesWithContributedCommits":0,"totalRepositoriesWithContributedPullRequests":0,"streak":{"current":{"objects":{},"id":-1,"startDate":"-","endDate":"-","totalDays":0,"totalContributions":0,"statisticId":0},"streaks":[]},"contributions":{"commit":{"total":0},"issue":{"total":0},"pullRequest":{"total":0},"pullRequestReview":{"total":0},"total":0},"busiestDay":{"id":-1,"date":"","total":0,"platformId":-1},"calendar":{"startDate":"2017-12-31T09:09:09.000Z","endDate":"2018-12-31T09:09:09.000Z","weeks":[{"days":[{"date":"2017-12-31","color":"#ebedf0","total":0},{"date":"2018-01-01","color":"#ebedf0","total":0},{"date":"2018-01-02","color":"#ebedf0","total":0},{"date":"2018-01-03","color":"#ebedf0","total":0},{"date":"2018-01-04","color":"#ebedf0","total":0},{"date":"2018-01-05","color":"#ebedf0","total":0},{"date":"2018-01-06","color":"#ebedf0","total":0}]},{"days":[{"date":"2018-01-07","color":"#ebedf0","total":0},{"date":"2018-01-08","color":"#ebedf0","total":0},{"date":"2018-01-09","color":"#ebedf0","total":0},{"date":"2018-01-10","color":"#ebedf0","total":0},{"date":"2018-01-11","color":"#ebedf0","total":0},{"date":"2018-01-12","color":"#ebedf0","total":0},{"date":"2018-01-13","color":"#ebedf0","total":0}]},{"days":[{"date":"2018-01-14","color":"#ebedf0","total":0},{"date":"2018-01-15","color":"#ebedf0","total":0},{"date":"2018-01-16","color":"#ebedf0","total":0},{"date":"2018-01-17","color":"#ebedf0","total":0},{"date":"2018-01-18","color":"#ebedf0","total":0},{"date":"2018-01-19","color":"#ebedf0","total":0},{"date":"2018-01-20","color":"#ebedf0","total":0}]},{"days":[{"date":"2018-01-21","color":"#ebedf0","total":0},{"date":"2018-01-22","color":"#ebedf0","total":0},{"date":"2018-01-23","color":"#ebedf0","total":0},{"date":"2018-01-24","color":"#ebedf0","total":0},{"date":"2018-01-25","color":"#ebedf0","total":0},{"date":"2018-01-26","color":"#ebedf0","total":0},{"date":"2018-01-27","color":"#ebedf0","total":0}]},{"days":[{"date":"2018-01-28","color":"#ebedf0","total":0},{"date":"2018-01-29","color":"#ebedf0","total":0},{"date":"2018-01-30","color":"#ebedf
```

Comment:
Can we run prettier on this file?

Reply:
I would leave it as compact as possible because formatting it would produce a lot of overhead.